tests/testthat/test_endpoints.R

```diff
@@ -0,0 +1,46 @@
+context("endpoints")
+
+test_that("endpoint_validate_baseline correctly validates data", {
+  pjnz <- system.file("testdata", "Botswana2018.PJNZ", package = "hintr")
+  response <- endpoint_validate_baseline("request", pjnz, NULL, NULL)
+  expected_response <- '{"status":"success","errors":{},"data":"Botswana"}'
+  expect_equal(as.vector(response), expected_response)
```

Comment:
Do you know if there is a better way for me to test the returned JSON? Casting to vector and comparing with a string seems a bit rubbish.